src/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+python_add_library(_fitting MODULE WITH_SOABI fit.cc fitting.cc)
+
+python_add_library(_functionals MODULE WITH_SOABI functional.cc functionals.cc)
+
+python_add_library(_images MODULE WITH_SOABI images.cc pyimages.cc)
+
+python_add_library(_measures MODULE WITH_SOABI pymeas.cc pymeasures.cc)
+
```

Comment:
Probably the wcslib and cfitsio linking should happen here, for the _images library.

Reply:
That happens further down in the `foreach` loop.